snap/local/enb.conf

```diff
@@ -0,0 +1,429 @@
+# report_json_filename: Report JSON filename (default: /tmp/enb_report.json)
+# report_json_asn1_oct: Prints ASN1 messages encoded as an octet string instead of plain text in the JSON report file
+# alarms_log_enable:    Enable Alarms logging (default: disabled)
+# alarms_filename:      Alarms logging filename (default: /tmp/alarms.log)
+# tracing_enable:       Write source code tracing information to a file
+# tracing_filename:     File path to use for tracing information
+# tracing_buffcapacity: Maximum capacity in bytes the tracing framework can store
+# stdout_ts_enable:     Prints once per second the timestamp into stdout
+# tx_amplitude:         Transmit amplitude factor (set 0-1 to reduce PAPR)
+# rrc_inactivity_timer  Inactivity timeout used to remove UE context from RRC (in milliseconds)
+# max_mac_dl_kos:       Maximum number of consecutive KOs in DL before triggering the UE's release (default: 100)
+# max_mac_ul_kos:       Maximum number of consecutive KOs in UL before triggering the UE's release (default: 100)
+# max_prach_offset_us:  Maximum allowed RACH offset (in us)
+# nof_prealloc_ues:     Number of UE memory resources to preallocate during eNB initialization for faster UE creation (default: 8)
+# rlf_release_timer_ms: Time taken by eNB to release UE context after it detects an RLF
+# eea_pref_list:        Ordered preference list for the selection of encryption algorithm (EEA) (default: EEA0, EEA2, EEA1)
+# eia_pref_list:        Ordered preference list for the selection of integrity algorithm (EIA) (default: EIA2, EIA1, EIA0)
+# gtpu_tunnel_timeout:  Time that GTPU takes to release indirect forwarding tunnel since the last received GTPU PDU (0 for no timer)
+# ts1_reloc_prep_timeout: S1AP TS 36.413 TS1RelocPrep Expiry Timeout value in milliseconds
+# ts1_reloc_overall_timeout: S1AP TS 36.413 TS1RelocOverall Expiry Timeout value in milliseconds
+# rlf_release_timer_ms: Time taken by eNB to release UE context after it detects a RLF
+# rlf_min_ul_snr_estim: SNR threshold in dB below which the enb is notified with RLF ko
+# s1_setup_max_retries: Maximum amount of retries to setup the S1AP connection. If this value is exceeded, an alarm is written to the log. -1 means infinity.
+# s1_connect_timer:     Connection Retry Timer for S1 connection (seconds)
+# rx_gain_offset:       RX Gain offset to add to rx_gain to calibrate RSRP readings
+#####################################################################
+[expert]
+#pusch_max_its        = 8 # These are half iterations
+#nr_pusch_max_its     = 10
+#pusch_8bit_decoder   = false
+#nof_phy_threads      = 3
+#metrics_period_secs  = 1
+#metrics_csv_enable   = false
+#metrics_csv_filename = /tmp/enb_metrics.csv
+#report_json_enable   = true
+#report_json_filename = /tmp/enb_report.json
+#report_json_asn1_oct = false
+#alarms_log_enable    = true
+#alarms_filename      = /tmp/enb_alarms.log
+#tracing_enable       = true
+#tracing_filename     = /tmp/enb_tracing.log
```

Comment:
Some more default paths that will not work

Reply:
Fixed.